chpl/chpl-service/src/main/java/gov/healthit/chpl/surveillance/report/builder/ComplaintsWorksheetBuilder.java

```diff
@@ -87,7 +87,7 @@ public ComplaintsWorksheetBuilder(ComplaintManager complaintManager,
         this.complaintManager = complaintManager;
         this.cpdManager = cpdManager;
         this.survDao = survDao;
-        dateFormatter = new SimpleDateFormat("MM/dd/yyyy");
+        dateFormatter = DateTimeFormatter.ofPattern("d MMMM yyyy");
```

Comment:
I don't think this line is necessary, since the `DateTimeFormatter` is initialized to the same value when the class is created.

Reply:
Removed the initialization during class creation in favor of initializing it in the constructor; the earlier part was there as an artifact of copy/paste